### ISSUE TITLE
Add-on results in error during SSR development

### ISSUE THREAD

**Issue by jeremy-code** [mozilla/fx-private-relay-add-on#514]
Hi, this issue isn't urgent, but I figured I might as well document it because it took some time for me to discover.

When running the development server for Next.js, and I suspect for other server-side rendered frameworks, I receive the error: 

```
Error: Hydration failed because the initial UI does not match what was rendered on the server.

Warning: Expected server HTML to contain a matching <input> in <div>.
```

The reasoning behind this error is because the client UI attempts to load in the Firefox Relay icon when using the Firefox Relay add-on, which conflicts with the server, resulting in a hydration error.

**Comment by Vinnl** [comment]
Yep, we've actually run into this ourselves as well. The easiest improvement to make would probably be to just not inject the icon on `localhost` at least. PRs welcome!

**Comment by pandashashwat97** [comment]
Hi Jeremy can you assign this issue to me, if its still open!!

**Comment by Vinnl** [comment]
@pandashashwat97 It's still open, feel free to submit a PR. [This](https://github.com/mozilla/fx-private-relay-add-on/blob/25d3de6184edac41452ce8ca4006a1032645a5c4/src/js/other-websites/icon.js#L10-L17) is probably where to looking; I'm guessing adding localhost to that list would resolve it, maybe including looking at the `hostname` instead of the `origin`.

**Comment by thelittleerik** [comment]
Ran into the same issue. Will this have any effect in the production build for users with Firefox Relay extension? 
Can I continue development without worrying about this issue?
